wikimedia/wikipedia-ios
# [ALT TEXT] Publish alt text for Experiment B

**Phabricator:** https://phabricator.wikimedia.org/T370236

### Notes
This PR adds the new alt text to the previous published article's wikitext and publishes again. Note that for now we are publishing when the "Next" button is selected on the half sheet alt text modal. When we complete the review screen we will move the logic to trigger after that screen's Publish button is tapped.

### Test Steps
1. To observe the general flow, fresh install, add a target wiki as your primary language and enable both the "Do not post image recommendations edit" and "Enable alt text experiment" feature flags. Accept an image recommendation, then go through the alt text flow when the modal appears (if you are in the 50%). Enter alt text and tap Next. Confirm the alt text views are popped, you see the "Edit published" toast and land on the next image recommendation.
2. To test wikitext publishing: reinstall, add a target wiki as your primary language, turn on the "Enable alt text experiment" feature flag but to NOT turn on the "Do not post image recommendations edit". If you want to test on EN, also turn on the "Enable alt text experiment on English Wikipedia" flag.
3. Accept an image recommendation and publish (note this is actually publishing).
4. If you are in the 50%, the alt text prompt will appear. Tap through and add alt text, then tap Next. (note this is actually publishing).
5. You should see the "Edit published" toast and land on the next image recommendation. Go to the article you published on Desktop wikipedia and check the edits.

One of my examples can be seen here: https://en.wikipedia.org/w/index.php?title=Gornja_Kravarica&action=history
